plugin.audio.radio_de/resources/lib/api.py

```diff
@@ -213,28 +302,41 @@ def __urlopen(self, url):
         return response
 
     @staticmethod
-    def __format_stations(stations):
+    def __format_stations_v2(stations):
         formated_stations = []
         for station in stations:
             thumbnail = (
-                station.get('picture4TransName') or
-                station.get('picture4Name') or
-                station.get('picture1TransName').replace('_1_', '_4_') or
-                station.get('picture1Name').replace('_1_', '_4_')
-            )
-            genre = station.get('genresAndTopics') or ','.join(
-                station.get('genres', []) + station.get('topics', []),
+                station.get('logo300x300') or
+                station.get('logo175x175') or
+                station.get('logo100x100') or
+                station.get('logo44x44')
             )
+
+            try:
```

Comment:
Could you not do `genre = [g.get('value', d) for g in station.get('genres')]`

Same for the descriptions and names